lib/exavier/mutators/negate_conditionals.ex

```diff
@@ -0,0 +1,59 @@
+defmodule Exavier.Mutators.NegateConditionals do
+  @moduledoc """
+  Mutates conditional operators into their opposite.
+
+  Conditionals are replaced according to the table below.
+
+  | Original | Mutation |
+  |----------|----------|
+  | ==       | !=       |
+  | !=       | ==       |
+  | <=       | >        |
+  | >=       | <        |
+  | <        | >=       |
+  | >        | <=       |
+
+  For example:
+
+      if a == b {
+        // do something
+      }
+
+  will be mutated into
+
+      if a != b {
+        // do something
+      }
+  """
```

Comment:
OMG these moduledocs. 😍

Reply:
I'm glad you like them! I like writing good docs :)

Reply:
Looks great! [Even better when published](https://hexdocs.pm/exavier/Exavier.Mutators.NegateConditionals.html) (as `0.2.0`) 🤩

Reply:
heck yeah!